### PR TITLE
Pypi release prep

### PR DESCRIPTION
**Pull request by rb201**
### Description:

**Replace with description of the changes**

### Issue Reference URL

https://github.com/awslabs/aws-eks-data-maker/issues/<NUMBER>

----------
### Do not change content below. Mark applicable check boxes only.

Thank you for submitting a contribution to AWS Orbit Workbench.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a ISSUE associated with this PR?
- [ ] Does your PR title start with ISSUE-XXXX where XXXX is the issue number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [ ] Has your PR been rebased against the latest commit within the target branch (typically 'main')?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under Apache License 2.0?

### CODE REVIEW

**Review by abaror — APPROVED** [review]

File: plugins/code_commit/setup.py [hunk 1]
```diff
@@ -22,10 +22,27 @@
...
     version=version,
+    author="AWS Professional Services",
+    author_email="aws-proserve-opensource@amazon.com",
+    url="https://github.com/awslabs/aws-orbit-workbench",
+    project_urls={"Org Site": "https://aws.amazon.com/professional-services/"},
     description="Orbit Workbench CodeCommit Plugin.",
     license="Apache License 2.0",
     packages=["code_commit"],
+    keywords=["aws", "cdk"],
     python_requires=">=3.6, <3.9",
```
Comment (chamcca) [inline plugins/code_commit/setup.py:33]: we'll need to fix these after the fact. a number of plugins list 3.6

File: plugins/custom_cfn/setup.py [hunk 2]
```diff
@@ -21,11 +21,28 @@
...
+    author="AWS Professional Services",
+    author_email="aws-proserve-opensource@amazon.com",
+    url="https://github.com/awslabs/aws-orbit-workbench",
+    project_urls={"Org Site": "https://aws.amazon.com/professional-services/"},
     version=version,
     description="Launch a CloudFormation stack for the team space",
+    keywords=["aws", "cdk"],
     license="Apache License 2.0",
     packages=find_packages(include=["custom_cfn", "custom_cfn.*"]),
     python_requires=">=3.6, <3.9",
```
Comment (chamcca) [inline plugins/custom_cfn/setup.py:33]: 3.6

File: plugins/code_commit/setup.py [hunk 3]
```diff
@@ -22,10 +22,27 @@
...
+        "Intended Audience :: Information Technology",
+        "Intended Audience :: Science/Research",
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python :: 3 :: Only",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Typing :: Typed",
+    ],
     install_requires=[
         "aws-cdk.aws-codecommit~=1.67.0",
```
Comment (chamcca) [inline plugins/code_commit/setup.py:47]: we should drop these cdk references from the install_requires as they'll be installed by our codebuild

File: plugins/hello_world/setup.py [hunk 4]
```diff
@@ -21,11 +21,28 @@
...
+        "Intended Audience :: Information Technology",
+        "Intended Audience :: Science/Research",
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python :: 3 :: Only",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Typing :: Typed",
+    ],
     install_requires=[
         "aws_cdk.core~=1.67.0",
```
Comment (chamcca) [inline plugins/hello_world/setup.py:47]: we'll drop these in the next release

**Pull request merged**